mnefun/_mnefun.py

```diff
@@ -1725,6 +1725,36 @@ def gen_forwards(p, subjects, structurals, run_indices):
             fwd = make_forward_solution(
                 info, trans, src, bem, n_jobs=p.n_jobs, mindist=p.fwd_mindist)
             write_forward_solution(fwd_name, fwd, overwrite=True)
+        # plot BEM & source space alignment
+        plt.rcParams['axes.titlesize'] = 'medium'
+        plt.rcParams['axes.titleweight'] = 'normal'
+        fontdict = dict(color='white',
+                        verticalalignment='top', horizontalalignment='center')
+        aln = mne.viz.plot_alignment(info, subjects_dir=subjects_dir,
+                                     bem=bem, src=src,
+                                     surfaces=['outer_skin', 'inner_skull'],
+                                     dig=True, coord_frame='meg',
+                                     show_axes=True)
+        aln.scene.parallel_projection = True
+        fig, axes = plt.subplots(1, 3, figsize=(6.5, 2.5), facecolor='k')
+        for ai, (angle, tv) in enumerate(zip([180, 90, 0], ['Left', 'Center',
+                                                            'right'])):
+            mlab.view(angle, 90, focalpoint=(0., 0., 0.), distance=0.6)
+            view = mlab.screenshot()
+            mask_w = (view == 0).all(axis=-1).all(axis=1)
+            mask_h = (view == 0).all(axis=-1).all(axis=0)
+            view = view[~mask_w][:, ~mask_h]
+            axes[ai].set_axis_off()
+            axes[ai].imshow(view, interpolation='bicubic')
+            axes[ai].set_title(tv, fontdict=fontdict)
+        mlab.close(aln)
+        fig.subplots_adjust(left=0, right=1, top=1, bottom=0, wspace=0.05,
+                            hspace=0)
+        fname = op.join(p.work_dir, subj, p.inverse_dir,
+                        op.basename(raw_fname)[:-4] +
+                        '_%s.png' % bem_type)
+        fig.savefig(fname, dpi=150, facecolor=fig.get_facecolor(),
+                    edgecolor='none')
```

Comment:
I don't really like the proliferation of all of these images. Why don't we just add all of them to the report? Is the report generation to slow or not useful enough?

Reply:
Will address comments. 

I am also not fond of the proliferation of images. Honestly, It's just me trying to get something done promptly and have just been holding off on messing with the report. IDK how useful others find it; I've asked the MEG team what if anything should be included in the report (or better yet open Github issues) wrt QC but crickets. 

I can start a PR to add:
1. cHPI SNR
2. PCA topomaps
3. Alignment
???

Reply:
That would be a great start, feel free to get one or more of those in a PR and we can do follow-up PRs for others we think of